packages/server/lib/modes/run.js

```diff
@@ -1117,14 +1117,14 @@ module.exports = {
   },
 
   runSpecs (options = {}) {
-    const { config, browser, sys, headed, outputPath, specs, specPattern, beforeSpecRun, afterSpecRun, runUrl, parallel, group, tag } = options
+    options = _.defaults(options, {
```

Comment:
`_.defaults(...)` mutates options, so let's not use a reassignment here since that signals the wrong thing